teuthology/task/install.py

```diff
@@ -926,6 +929,105 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
         ] + ['%s=%s' % (d, version) for d in debs],
     )
 
+@contextlib.contextmanager
+def rh_install(ctx,config):
+    """
+    Installs rh ceph on all hosts in ctx.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict 
+    """
+    with parallel() as p:
+        for remote in ctx.cluster.remotes.iterkeys():
+            distro = teuthology.get_system_type(remote,distro=True)
+            if distro == 'redhatenterpriseserver':
```

Comment:
Instead of the above, please use ``remote.os.name`` - it will be 'rhel' for RHEL nodes.